### ISSUE TITLE
Extract the mask of the entire image

### ISSUE THREAD

**Issue by mowangmodi** [rover-xingyu/Ha-NeRF#35]
Hello Xingyu, may I ask how to extract the mask of the entire image based on the pre trained model you provided?

**Comment by mowangmodi** [comment]
The pre trained model you provided does not have transient embedded MLP weights

**Comment by rover-xingyu** [comment]
Yes, the pre-trained model does not include transient embedded MLP weights, but they can be easily obtained by training using our provided training code.

**Comment by mowangmodi** [comment]
> 是的，预训练模型不包括瞬态嵌入式 MLP 权重，但可以使用我们提供的训练代码轻松获得它们。

Can I load the pre trained model you provided in the test code to obtain the mask of the entire image?

**Comment by mowangmodi** [comment]
Or can I only load the pre trained model you provided in the training code to obtain the mask of the entire image? Do I also need to reinitialize the transient embedded network, as the pre trained model does not have MLP weights
